installation.md

```diff
@@ -7,7 +7,7 @@ description: Create a new Laravel Zero project
 
 > **Requires:**
 >- [PHP 8.1+](https://php.net/releases)
->- [`sodium` PHP extension](https://php.net/manual/book.sodium.php) (required by [Box](https://github.com/box-project/box)) - if you are bundling to a Phar
+>- [__sodium__ ](https://php.net/manual/book.sodium.php) PHP extension (required by [Box](https://github.com/box-project/box)) - if you are bundling to a Phar
```

Comment:
I'm wondering if it may look best with a mix of both?
```suggestion
>- [**`sodium`** PHP extension](https://php.net/manual/book.sodium.php) (required by [Box](https://github.com/box-project/box)) - if you are bundling to a Phar
```